flow/scripts/variables.yaml

```diff
@@ -442,17 +446,23 @@ ADDITIONAL_FILES:
   - grt
   - route
   - final
-ADDITIONAL_LEFS:
-  description: >
-    Hardened macro LEF view files listed here. The LEF information of the macros
-    is immutable and used throughout all stages. Stored in the .odb file.
 ADDITIONAL_LIBS:
   description: >
     Hardened macro library files listed here. The library information is immutable
     and used throughout all stages. Not stored in the .odb file.
+  stages:
+  - synth
+  - floorplan
+  - place
+  - cts
+  - grt
+  - route
+  - final
```

Comment:
Is there a more succinct way to say 'all'?  ADDITIONAL_LEFS lists no stages which I assume is equivalent though less obvious (is it an oversight or all).

Reply:
We haven't added that and I have been dithering on whether it is a good idea, it adds a tiny bit complication for a marginal benefit. Now we just leave it blank if we don't care enough to distinguish exactly when it is used and that or it must always be included.

The above is not "all", it is specifically all minus generate_abstract.

Reply:
Ok then it is wrong as write_abstract_lef depends on timing.

Reply:
*additional_libs* are not needed to write the abstract .lib file today, is it?

Could it be in the future?

(I'll add them, just checking for clarification).

Reply:
I see now that additional_libs is read used in generate_abstract. Removing specific stages for additional_libs, they are used pervasively...